### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by injectives**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+| 4.3 | 8, 11 | Primary development branch. | [link](https://github.com/neo4j/neo4j-java-driver/wiki/4.3-changelog) |
+| 4.2 | 8, 11 | Maintenance. | [link](https://github.com/neo4j/neo4j-java-driver/wiki/4.2-changelog) |
+| 4.1 | 8, 11 | Maintenance. | [link](https://github.com/neo4j/neo4j-java-driver/wiki/4.1-changelog) |
 
-### Java Runtime
+## Server Compatibility
 
-Recent drivers require the Java 8 or higher version of the runtime.
-The table below shows runtime compatibility for the currently-supported driver versions.
+The compatibility with Neo4j Server versions is documented in the [Neo4j Knowledge Base](https://neo4j.com/developer/kb/neo4j-supported-versions/).
```
Comment (bigmontz) [inline README.md:19]: Great addition

File: README.md [hunk 2]
```diff
@@ -78,24 +76,22 @@ when raising an issue at [Issues](https://github.com/neo4j/neo4j-java-driver/iss
 When reporting, please mention the versions of the driver and server, as well as the server topology (single instance, causal cluster, etc).
 Also include any error stacktraces and a code snippet to reproduce the error if possible, as well as anything else that you think might be helpful.
 
-
-## For Driver Developers
+## For Driver Engineers
```
Comment (bigmontz) [inline README.md:79]: Fancy

File: README.md (hunk 1 above)
Comment (injectives) [reply README.md:19]: Yeah, might be useful :)

**Review by bigmontz — APPROVED** [review]
🌵

File: README.md (hunk 2 above)
Comment (injectives) [reply README.md:79]: 👍

**Pull request merged**